op-node/flags/flags.go

```diff
@@ -182,8 +182,12 @@ var (
 	L1CacheSize = &cli.UintFlag{
 		Name: "l1.cache-size",
 		Usage: "Cache size for blocks, receipts and transactions. " +
-			"It's optional and a sane default of 3/2 the sequencing window size is used if this field is set to 0.",
+			"If this flag is set to 0, 2/3 of the sequencing window size is used. " +
+			"The default value of 900 (~3h of L1 blocks) is good for (high-throughput) networks that see frequent safe head increments. " +
+			"On (low-throuhgput) networks with infrequent safe head increments, it is recommended to set this value to 0, " +
+			"or a value that well covers the typical span between safe head increments.",
 		EnvVars:  prefixEnvVars("L1_CACHE_SIZE"),
+		Value:    900, // ~3h of L1 blocks
```

Comment:
It seems better/safer to me to default to the low-throughput of 2400 blocks, which also works for high-throughput chains. Then high-throughput chains could adjust that lower as an optimization.

Also a little confusing to me to express this setting in different terms: 2/3 sequencer window instead of an L1 block amount as is used for the "Value". I'd prefer to say 2400 instead of 2/3 seq window

Reply:
The purpose of this PR was to _lower_ the default. If we wanted to keep it at 2400, we could just close this PR 😅 

"2/3 seq window" is more general than 2400, because chains can be configured to use different seq window sizes, although this is very uncommon.

Most node ops are actually running nodes for high-throughput chains, so when they upgraded, including us, they/we saw an unexpected rise in memory going over 4GB. So it's more optimal to have a lower default memory consumption in op-geth and only have low-through chain nodes explicitly raise the cache.

Reply:
added a bit of clarification to the flag description